public/robots.txt

```diff
@@ -2,6 +2,8 @@
 #
 # To ban all spiders from the entire site uncomment the next two lines:
  User-agent: *
- Disallow: /flags
+ Disallow: /admins
+ Disallow: /blockeds
+ Disallow: /activities
 
  Sitemap: http://www.catforum.mntc.in/sitemap.xml
```

Comment:
If you are sure that blocked controller is not doing anything then you delete it. Correct process would be roll back migration file of blocked and then `rails d scaffold Blocked`